doc/pages/keys.asciidoc

```diff
@@ -681,6 +681,9 @@ in order to specify the wanted object:
 *c*::
     select user defined object, will prompt for open and close text
 
+*<a-;>*::
+    run a command
```

Comment:
I find this description a bit too much straight to the point. By reading this, I would think "why would I need to run a command here?"

Reply:
updated